chainerrl/wrappers/__init__.py

```diff
@@ -4,3 +4,13 @@
 from chainerrl.wrappers.randomize_action import RandomizeAction  # NOQA
 
 from chainerrl.wrappers.scale_reward import ScaleReward  # NOQA
+
+from chainerrl.wrappers.atari_wrappers import LazyFrames  # NOQA
+from chainerrl.wrappers.atari_wrappers import ScaledFloatFrame  # NOQA
+from chainerrl.wrappers.atari_wrappers import FrameStack  # NOQA
+from chainerrl.wrappers.atari_wrappers import WarpFrame  # NOQA
+from chainerrl.wrappers.atari_wrappers import ClipRewardEnv  # NOQA
+from chainerrl.wrappers.atari_wrappers import MaxAndSkipEnv  # NOQA
+from chainerrl.wrappers.atari_wrappers import EpisodicLifeEnv  # NOQA
+from chainerrl.wrappers.atari_wrappers import FireResetEnv  # NOQA
+from chainerrl.wrappers.atari_wrappers import NoopResetEnv  # NOQA
```

Comment:
Some of these wrappers are specific to atari.  I'd rather not make them available under `chainerrl.wrappers`.